dataproc_jupyter_plugin/commons/constants.py

```diff
@@ -29,3 +29,4 @@
 COMPUTE_SERVICE_DEFAULT_URL = "https://compute.googleapis.com/compute/v1"
 STORAGE_SERVICE_DEFAULT_URL = "https://storage.googleapis.com/storage/v1/"
 WRAPPER_PAPPERMILL_FILE = "wrapper_papermill.py"
+DIRECTORY = "./Scheduled-jobs"
```

Comment:
Rename the variable to LOCAL_DAG_FILE_LOCATION

Reply:
completed